packages/esm-appointments-app/src/helpers/excel.ts

```diff
@@ -13,14 +13,17 @@ export function downloadAppointmentsAsExcel(
     time: true,
   })}`,
 ) {
-  const appointmentsJSON = appointments?.map(({ name, gender, age, serviceType, dateTime, phoneNumber }) => ({
-    'Patient name': name,
-    Gender: gender === 'F' ? 'Female' : 'Male',
-    Age: age,
-    'Appointment type': serviceType,
-    Date: formatDate(new Date(dateTime), { mode: 'wide' }),
-    'Phone Number': phoneNumber || '--',
-  }));
+  const appointmentsJSON = appointments?.map(
+    ({ name, gender, age, serviceType, dateTime, phoneNumber, identifier }) => ({
+      'Patient name': name,
+      Gender: gender === 'F' ? 'Female' : 'Male',
+      Age: age,
+      'CCC Number': identifier,
```

Comment:
Should this have the title `CCC` or identifier

Reply:
identifier will do